tools/serverpod_cli/test/integration/analyzer/dart/endpoint_validation/endpoint_method_test.dart

```diff
@@ -120,17 +120,21 @@ class ExampleEndpoint extends Endpoint {
     });
   });
 
-  group('Given an endpoint method without Session param when analyzed', () {
+  group('Given an endpoint method without a first unnamed `Session` param', () {
```

Comment:
Suggestion(NIT): Change to `first positional param`.

In dart, the unnamed parameters are reffered to as positional parameters: https://dart.dev/language/functions#parameters

Reply:
Great, thanks! 🙌

Reply:
[7c32c17](https://github.com/serverpod/serverpod/pull/2597/commits/7c32c179843f7b45b4b4f6b77cb51d76e26568bc)